code/workspaces/web-app/src/components/stacks/StackCardActions.js

```diff
@@ -48,7 +48,7 @@ export const PureStackCardActions = ({ stack, openStack, deleteStack, editStack,
 
   return (
     <div className={classes.cardActions}>
-      {openStack && <PermissionWrapper className={classes.buttonWrapper} userPermissions={userPermissions} permission={openPermission}>
+      {openStack && stack.status === READY && <PermissionWrapper className={classes.buttonWrapper} userPermissions={userPermissions} permission={openPermission}>
         <PrimaryActionButton
           disabled={!isReady(stack)}
```

Comment:
I don't know whether you should have the `stack.status === READY` check here. If I'm reading this properly then unless the stack is ready, the open button won't be displayed. I think as the changes are being made to always have the options button being displayed then it would be a good idea to always have the open button displayed as well but make disabled until the stack is ready. If you go that way then it is probably worth adding a tool tip to the open button that says the stack can't be opened until it is ready.

Reply:
I think you're right - have changed this now